### PR TITLE
Relayer proof

### PR DESCRIPTION
**Pull request by evgeny-stakewise**

### CODE REVIEW

File: src/validators/relayer.py [hunk 1]
```diff
@@ -30,16 +32,25 @@ async def get_validators(self, start_index: int, count: int) -> RelayerValidator
...
+                    amount_gwei=validator_item['amount_gwei'],
+                    signature=validator_item['deposit_signature'],
+                    exit_signature=BLSSignature(
+                        Web3.to_bytes(hexstr=validator_item['exit_signature'])
+                    ),
+                )
+                validators.append(validator)
+
+            multi_proof: MultiProof[tuple[bytes, int]] | None = None
+            if resp_json['proof'] is not None:
```
Comment (tsudmi) [inline src/validators/relayer.py:48]:
```suggestion
            if resp_json.get('proof', None) is not None:
```

File: src/validators/relayer.py [hunk 2]
```diff
@@ -30,16 +32,25 @@ async def get_validators(self, start_index: int, count: int) -> RelayerValidator
...
+                leaves = list(zip(tx_validators, resp_json['proof_indexes']))
+                multi_proof = MultiProof(
+                    leaves=leaves, proof=resp_json['proof'], proof_flags=resp_json['proof_flags']
                 )
-                for v in resp_json['validators']
-            ]
             return RelayerValidatorsResponse(
-                validators=validators,
-                validators_manager_signature=resp_json['validators_manager_signature'],
+                validators=validators, validators_manager_signature=None, multi_proof=multi_proof
```
Comment (tsudmi) [inline src/validators/relayer.py:55]: Why validators manager signature is always None?

**Comment by evgeny-stakewise** [comment]
Deposit data should be in located in Operator in DVT case. Set PR on pause.

File: src/validators/relayer.py (hunk 1 above)
Comment (evgeny-stakewise) [reply src/validators/relayer.py]: relayer.py was reworked

File: src/validators/relayer.py (hunk 2 above)
Comment (evgeny-stakewise) [reply src/validators/relayer.py]: reworked

File: src/validators/execution.py [hunk 3]
```diff
@@ -197,3 +197,11 @@ async def update_unused_validator_keys_metric(
     metrics.unused_validator_keys.set(validators)
 
     return validators
+
+
+async def get_validators_start_index() -> int:
```
Comment (evgeny-stakewise) [inline src/validators/execution.py:202]: Moved from src/validators/tasks.py

**Review by cyc60 — APPROVED** [review]

**Pull request merged**